src/pages/SignUp/Components/SignUpAfter.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import * as S from './SignUpAfter.styled';
+
+const SignUpAfter = () => {
+  const [disable, setDisable] = useState(false);
+  const [userInfo, setUserInfo] = useState({
+    id: '',
+    name: '',
+    email: '',
+  });
+  const navigate = useNavigate();
+  const handleUserInfo = e => {
+    const { name, value } = e.target;
+    setUserInfo(before => ({ ...before, [name]: value }));
+  };
+  const handleSubmit = () => {
+    fetch('http://10.58.7.122:8000/users/kakao', {
+      method: 'POST',
+      body: JSON.stringify({
+        kakao_id: userInfo.id,
+        user_name: userInfo.name,
+        email: userInfo.email,
+      }),
+    });
+    navigate('/sign-in');
+  };
+  const getProfile = async () => {
+    try {
+      let data = await window.Kakao.API.request({
+        url: '/v2/user/me',
+      });
+      console.log(data);
+      setUserInfo(before => ({
+        ...before,
+        id: data.id,
+        name: data.properties.nickname,
+        email: data.kakao_account.email,
+      }));
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  useEffect(() => {
+    getProfile();
+  }, []);
+  return (
+    <S.SignUpAfterContainerBox>
+      <S.SignUpAfterContainer>
+        <S.Text fontSize={40} fontWeight="bold">
+          회원가입
+        </S.Text>
+        <S.Text>최소한의 정보를 받고있습니다.</S.Text>
+        <S.UserInfoContainer>
+          <S.Text>이름</S.Text>
+          <S.Input
+            name="name"
+            placeholder={userInfo.name ? userInfo.name : '이름'}
```

Comment:
논리합 연산자를 활용할 수 있겠네요!
```suggestion
            placeholder={userInfo.name || '이름'}
```

Reply:
논리합 연산자를 활용하도록 수정하였습니다!